specs/Standards/MultiToken/Core.md

```diff
@@ -24,6 +24,21 @@ Prior art:
 - [ERC-1155]
 - [NEAR Fungible Token Standard][FT], which first pioneered the "transfer and call" technique
 - [NEAR Non-Fungible Token Standard][NFT]
+## Rationale
+
+Why have another standard, aren't fungible and non-fungible tokens enough?  The current fungible token and non-fungible token standards, do not provide support for performing batch transactions, representing many tokens in a single contract, as well as the flexibility to define different token types with different behavior in a single contract. This is something that makes it impossible to be interoperable with other major blockchain networks such as Ethereum.
```

Comment:
The critic here is not entirely valid.

> do not provide support for performing batch transactions

Are you sure? Even so, it could be solved easily by yet another extension without breaking compatibility. We don't need the new standard because of that.

> representing many tokens in a single contract

It's better to clarify: this concern is only about FT. We can also mention "do not provide the ability to specify the amount" about NFT.

> This is something that makes it impossible to be interoperable with other major blockchain networks such as Ethereum.

You are talking here about ERC-1155 compatibility, but it sounds like our FT/NFT could not be represented on the ETH side. They actually can (ERC-20, ERC-721).